packages/ui/components/input-stepper/src/LionInputStepper.js

```diff
@@ -265,8 +265,19 @@ export class LionInputStepper extends LocalizeMixin(LionInput) {
    * @private
    */
   __increment() {
-    const { step, min, max } = this.values;
-    const newValue = this.currentValue + step;
+    const { step, max } = this.values;
+    let { min } = this.values;
+    if (min === Infinity) {
```

Comment:
Can we align both functions?

I prefer the `const stepMin = min !== Infinity ? min : 0;` approach, thats more readable.